### ISSUE TITLE
Not allowed to save q47

### ISSUE THREAD

**Issue by ClaraDOAJ** [DOAJ/doaj#1351]
![image](https://user-images.githubusercontent.com/22593616/28516489-b7e2d6d8-7061-11e7-96af-671ab7a0799e.png)

We are experiencing some problems with q47. Apparently this has happened before but was solved by
removing any options selected in q48 (that appears if you select Other in q47).

This time there were/are no options in 48 and as the journal has got CC BY.

Could you please have a look and see if this is performing correctly?

Thanks!

**Comment by emanuil-tolev** [comment]
Are you editing a journal or application that should be save-able, but is not? Because q47 (licencing) appears to be empty?

Which journal / application are you seeing this on?

**Comment by ClaraDOAJ** [comment]
It is: Naukovì Vìstì Nacìonalʹnogo Tehnìčnogo Unìversitetu Ukraïni Kiïvsʹkij Polìtehnìčnij Institut
issn: 1810-0546

**Comment by Steven-Eardley** [comment]
For info:
License information on the record for that application -

```json
"license": [
                {
                    "open_access": true,
                    "embedded": false,
                    "title": " Creative Commons Attribution License",
                    "url": "",
                    "embedded_example_url": "",
                    "type": " Creative Commons Attribution License"
                }
            ],
```

A minor anomaly here is that there's a space before the license name, but I see the license name when I open the record:

https://doaj.org/admin/suggestion/fcf7f6a0c6ed4eefaba183618da0648c

<img width="539" alt="screen shot 2017-07-24 at 19 58 07" src="https://user-images.githubusercontent.com/1446355/28539777-ca3cbfd0-70aa-11e7-8158-5423310fb9c0.png">

**Comment by Steven-Eardley** [comment]
So it looks like the license record is the wrong shape, meaning the form can't parse question 48 from it.

**Comment by ClaraDOAJ** [comment]
Thanks, is there an easy solution to this?

**Comment by emanuil-tolev** [comment]
Hmm, the two screenshots are quite different. When I open the record I get Steve's screenshot - q47 is filled in and is not a problem. In the original post by Clara, q47 *is* the problem.

In fact, as it stands, this application record seems perfectly fine to save. I even tried saving the same record on test (which has the same licence data) and it worked. http://testdoaj.cottagelabs.com/admin/suggestion/fcf7f6a0c6ed4eefaba183618da0648c

> So it looks like the license record is the wrong shape, meaning the form can't parse question 48 from it.

What do you mean by that @Steven-Eardley? q48 is optional, so it seems entirely possible that all q48 checkboxes should be empty - they don't have to be provided even if "Other" was selected in q47.

* * *

To recap, so far it seems that somebody went and entered `[SPACE]Creative Commons Attribution License` in the Other field in q47. They did not answer q48 which would have popped up. The form is parsing everything correctly and allows saving.

Let's talk about it today at our meeting and try to reproduce Clara's original problem, where q47 appeared to be empty and prevented form saving.

**Comment by emanuil-tolev** [comment]
Problem appears to have gone away. Let's screenshare with the user if it happens again.
